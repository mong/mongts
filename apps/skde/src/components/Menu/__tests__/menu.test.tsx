/**
 * To view a rendered version of all the snapshots:
 *
 * Copy the contents of ./__snapshots__/index.tsx.snap
 * Go to https://codesandbox.io/s/snapshot-viewer-lqjjsm?file=/src/snapshots.js
 * Replace the contents with what you copied
 */

import { render } from "@testing-library/react";
import { Menu } from "..";

test("English menu", async () => {
  const { container } = render(<Menu lang="en" />);
  expect(container).toMatchSnapshot();
});

test("Norwegian menu", async () => {
  const { container } = render(<Menu lang="no" />);
  expect(container).toMatchSnapshot();
});
