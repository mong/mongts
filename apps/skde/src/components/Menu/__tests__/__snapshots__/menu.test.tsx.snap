// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`English menu 1`] = `
<div>
  <div
    class="_main_menu_29d207"
    data-testid="mainMenu"
  >
    <div
      class="_container_29d207"
    >
      <div
        class="_nav_group_29d207"
      >
        <div
          class="_body_29d207"
        >
          <div
            class="_inner_nav_group_29d207"
          >
            <ul>
              <span
                class="_title_29d207"
              >
                About
              </span>
              <ul>
                <li>
                  <a
                    href="/helseatlas/en/static/about"
                  >
                    Helseatlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/static/contact"
                  >
                    Contact
                  </a>
                </li>
              </ul>
            </ul>
          </div>
          <div
            class="_inner_nav_group_29d207"
          >
            <ul>
              <span
                class="_title_29d207"
              >
                Our Helseatlas
              </span>
              <ul>
                <li>
                  <a
                    href="/helseatlas/en/v2/kronikere"
                  >
                    Health atlas for chronic diseases
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/kvalitet"
                  >
                    Healthcare Quality Atlas
                  </a>
                </li>
                <li
                  data-testid="menuAtlasLink1"
                >
                  <a
                    href="/helseatlas/en/v2/psyk"
                  >
                    Healthcare Atlas for Mental Healthcare and Substance Abuse Treatment
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/fodsel"
                  >
                    Obstetrics Healthcare Atlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/gyn"
                  >
                    Gynaecology Healthcare Atlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/ortopedi"
                  >
                    Orthopaedic Healthcare Atlas for Norway
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/dagkir2"
                  >
                    Day surgery in Norway 2013–2017
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/kols"
                  >
                    COPD Healthcare Atlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/eldre"
                  >
                    Healthcare Atlas for the Elderly in Norway
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/nyfodt"
                  >
                    The Norwegian Neonatal Healthcare Atlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/barn"
                  >
                    Child healthcare atlas for Norway
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/en/v2/dagkir"
                  >
                    Day surgery in Norway 2011–2013
                  </a>
                </li>
              </ul>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Norwegian menu 1`] = `
<div>
  <div
    class="_main_menu_29d207"
    data-testid="mainMenu"
  >
    <div
      class="_container_29d207"
    >
      <div
        class="_nav_group_29d207"
      >
        <div
          class="_body_29d207"
        >
          <div
            class="_inner_nav_group_29d207"
          >
            <ul>
              <span
                class="_title_29d207"
              >
                Om oss
              </span>
              <ul>
                <li>
                  <a
                    href="/helseatlas/statisk/om"
                  >
                    Om helseatlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/statisk/planer-for-helseatlastjenesten"
                  >
                    Planer for helseatlastjenesten
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/statisk/kontakt"
                  >
                    Kontakt
                  </a>
                </li>
              </ul>
            </ul>
          </div>
          <div
            class="_inner_nav_group_29d207"
          >
            <ul>
              <span
                class="_title_29d207"
              >
                Våre helseatlas
              </span>
              <ul>
                <li
                  data-testid="menuAtlasLink2"
                >
                  <a
                    href="/helseatlas/v2/radiologi"
                  >
                    Helseatlas radiologi første del, MR
                  </a>
                </li>
                <li
                  data-testid="menuAtlasLink2"
                >
                  <a
                    href="/helseatlas/v2/kronikere"
                  >
                    Helseatlas for utvalgte kroniske sykdommer
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/kvalitet"
                  >
                    Helseatlas for kvalitet
                  </a>
                </li>
                <li
                  data-testid="menuAtlasLink1"
                >
                  <a
                    href="/helseatlas/v2/psyk"
                  >
                    Helseatlas for psykisk helsevern og rusbehandling
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/fodsel"
                  >
                    Helseatlas for fødselshjelp
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/gyn"
                  >
                    Helseatlas for gynekologi
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/ortopedi"
                  >
                    Helseatlas i ortopedi for Noreg
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/dagkir2"
                  >
                    Dagkirurgi i Norge 2013–2017
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/kols"
                  >
                    Helseatlas kols
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/eldre"
                  >
                    Eldrehelseatlas for Norge
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/nyfodt"
                  >
                    Norsk nyfødtmedisinsk helseatlas
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/barn"
                  >
                    Barnehelseatlas for Norge
                  </a>
                </li>
                <li>
                  <a
                    href="/helseatlas/v2/dagkir"
                  >
                    Dagkirurgi i Norge 2011–2013
                  </a>
                </li>
              </ul>
            </ul>
          </div>
          <div
            class="_inner_nav_group_29d207"
          >
            <ul>
              <span
                class="_title_29d207"
              >
                Andre analyser
              </span>
              <ul>
                <li
                  data-testid="menuAtlasLink2"
                >
                  <a
                    href="/helseatlas/v2/endometriose"
                  >
                    Kirurgiske inngrep for endometriose
                  </a>
                </li>
              </ul>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
